example/android/app/build.gradle

```diff
@@ -45,10 +45,10 @@ android {
             signingConfig signingConfigs.debug
         }
     }
+
```

Comment:
remove this change

Reply:
I tough it would be more common to add that line, but I personally like it more too to remove it. We might should think about a common format to avoid this.